Fix bug #235632: Apparently this code is still triggered on 64-bit systems
through configure, when configure is dealing with the proper LFS64 stuff
already (or should allow the glibc wrapper to do so). -JeR

--- libncftp/ncftp.h.orig	2008-08-25 18:55:38.000000000 +0200
+++ libncftp/ncftp.h	2008-08-25 19:09:02.000000000 +0200
@@ -165,17 +165,6 @@
 #	endif
 #else
 	/* Includes for UNIX */
-#	if ((defined(__linux__)) && (! defined(_LARGEFILE64_SOURCE)) && (! defined(NO_LARGEFILE64_SOURCE)) && (! defined(_LARGEFILE_SOURCE)) && (! defined(_FILE_OFFSET_BITS)))
-		/* Define _LARGEFILE64_SOURCE or NO_LARGEFILE64_SOURCE to
-		 * quiet this warning.  Be sure to define this for ALL
-		 * code that uses LibNcFTP.
-		 */
-#		warning "<ncftp.h> is #defining _LARGEFILE64_SOURCE for you"
-#		define _LARGEFILE64_SOURCE 1
-#		define   Stat    stat64
-#		define   Lstat   lstat64
-#		define   Fstat   fstat64
-#	endif
 #	include <unistd.h>
 #	include <sys/types.h>
 #	include <sys/stat.h>
